scripts/storybook/webpack.config.js

```diff
@@ -86,5 +89,33 @@ module.exports = (/** @type {webpack.Configuration} */ config) => {
 
   config.optimization.minimize = false;
 
+  if (process.env.TF_BUILD) {
+    console.log(
+      'Storybook webpack config:',
+      // Plugins have circular references, and they're probably not the issue here
+      JSON.stringify(config, (key, value) => (key === 'plugins' ? undefined : value), 2),
+    );
+    const northstarPath = path.join(gitRoot, 'packages/fluentui/react-northstar');
+    const filesToWatch = [
+      path.join(gitRoot, 'tsconfig.json'),
+      path.join(northstarPath, 'tsconfig.json'),
+      path.join(northstarPath, 'package.json'),
+    ];
+    for (const file of filesToWatch) {
+      fs.watchFile(file, (curr, prev) => {
```

Comment:
can we clean up watches when done via `fs.unwatchFile` ?

also recommended approach is to use `fs.watch()` -> https://nodejs.org/api/fs.html#fs_fs_watchfile_filename_options_listener

Reply:
The watches are meant to last until the processes is terminated. Is manually unwatching still necessary in that case?

From the docs it looked like `fs.watch()` wouldn't be notified when the file is *accessed*, which is what I need.

Reply:
>  Is manually unwatching still necessary in that case?

I dunno, if those will be properly cleaned up/GC all good